### PR TITLE
Update doc for ip_rules of storage account

### PR DESCRIPTION
**Pull request by neil-yechenwei**
fixes terraform-providers/terraform-provider-azurerm#2367

### CODE REVIEW

**Comment by neil-yechenwei** [comment]
Hopes this [PR ](https://github.com/terraform-providers/terraform-provider-azurerm/pull/7660)would be helpful.

File: website/docs/r/storage_account.html.markdown [hunk 1]
```diff
@@ -215,6 +215,8 @@ any combination of `Logging`, `Metrics`, `AzureServices`, or `None`.
 
 ~> **NOTE:** Network Rules can be defined either directly on the `azurerm_storage_account` resource, or using the `azurerm_storage_account_network_rules` resource - but the two cannot be used together. If both are used against the same Storage Account, spurious changes will occur. When managing Network Rules using this resource, to change from a `default_action` of `Deny` to `Allow` requires defining, rather than removing, the block.
 
+~> **Note:** The prefix of `ip_rules` must be between 0 and 30. `ip_rules` support public IP addresses only.
```
Comment (katbyte) [inline website/docs/r/storage_account.html.markdown:218]:
```suggestion
~> **Note:** The prefix of `ip_rules` must be between 0 and 30 and only supports public IP addresses.
```

Comment (neil-yechenwei) [reply website/docs/r/storage_account.html.markdown]: thx. updated

**Comment by neil-yechenwei** [comment]
@katbyte , THanks for your comments. I've updated code.

**Pull request merged**